packages/modules/localstack/src/localstack-container.ts

```diff
@@ -5,6 +5,8 @@ export const LOCALSTACK_PORT = 4566;
 export class LocalstackContainer extends GenericContainer {
   constructor(image = "localstack/localstack:2.2.0") {
     super(image);
+    this.resolveHostname();
```

Comment:
@cristianrgreco This line should not have been moved. There is no way to provide an environment before the constructor is called so `resolveHostname` will never resolve `LOCALSTACK_HOST`